<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineeringDialog</class>
 <widget class="QWidget" name="EngineeringDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>811</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>681</width>
    <height>811</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>681</width>
    <height>811</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>viewOptions</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">/* CSS Styles used throughout the Avinger code base. Some minor variations may exist. */

/* Pushbuttons: */

QWidget {
	font: 18px DinPRO-medium;
    color: white;
}

QWidget#viewOptions {
	border-radius: 5px;
	background: black;
}


/****************
 *         v3.0            *
 ****************/


/* --------- General font and layout defaults ----------*/

QDialog {
 font: 18px &quot;DinPRO-regular&quot;;
 color: white;
 background-color: rgb( 147, 147, 147 );
 border: 1px solid rgb( 147, 147, 147 );
 border-radius: 5px;
}

QWidget {
 font: 13px &quot;DinPRO-regular&quot;;
}

QLabel {
 font: 18px &quot;DinPRO-regular&quot;;
}

QLabel#titlebarLabel {
	background: rgb( 51, 51, 51 );
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
}
QLabel#titlebarTextLabel {
    font: 16pt &quot;DINPro-Medium&quot;;
    color: white;
}

QLabel#instructionsLabel
{
    background: rgb( 191, 191, 191 );
}

QLabel#instructionsTextLabel
{
    font: 12pt &quot;DINPro-Medium&quot;;
}

QLabel#wideDivideLabel
{
    background: rgb( 191, 191, 191 );
}

/* ---------------- Pushbuttons ----------------*/

QPushButton 
{
    background: rgb( 52, 73, 94 );
    border: 1px solid rgb( 52, 73, 94 );
    color: white;
    font: 10pt &quot;DinPRO-Medium&quot;;
    text-align: center;
    border-radius: 10px;
    border-width: 2px;
}

QPushButton::hover 
{
    background: rgb(52, 73, 94 );
    border-color: rgb( 166, 166, 166 );
    color: rgb( 214, 214, 214);
    border-width: 1px;
    border-radius: 10px;
}

 QPushButton:default 
{
     background: rgb( 51, 51, 51 );
/*     border: 1px solid black; /* make the default button prominent */
/*	 font: 17px &quot;DinPRO-bold&quot;;*/
 }

QPushButton:disabled 
{
     background:  rgb( 51, 51, 51 ); 
    border: rgb( 51, 51, 51 ); 
 }

/* --------- Spin Boxes -----------*/

QAbstractSpinBox {
	font: 18px DinPRO-medium;
	border: 0;
    border-radius: 5px;
}

QAbstractSpinBox::up-button {
	border: 0;
   background: rgba(233,233,235,255);
   border-top-right-radius: 8px;
   margin-top: -2px;
   margin-bottom: -1px;
   padding-bottom: -5px;
   width: 15px;
}

QAbstractSpinBox::up-arrow  {
  image: url(:/images/arrow_up_normal.png);
}

QAbstractSpinBox::up-arrow:hover {
  image: url(:images/arrow_up_rollover.png);
}

QAbstractSpinBox::down-button {
   border: 0;
   background: rgba(233,233,235,255);
   border-bottom-right-radius: 8px;
   padding-top: -5px;
   margin-bottom: -2px;
   margin-top: -1px;
   width: 15px;
}

QAbstractSpinBox::down-arrow:hover {
  image: url(:/images/arrow_down_rollover.png);
}

QAbstractSpinBox::down-arrow {
  image: url(:/images/arrow_down_normal.png);
}

/* ---------- Group Boxes ---------- */

 QGroupBox {
    font: 20px DINPro-medium;
    color: rgba(210, 210, 210, 255);
     background-color:    rgb( 128, 128, 128 );
/*     border: 2px solid #A0A0A0;*/
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top left; /* position at the top center */
     left: 5px;
     padding: 0 3px;
     color: white;
 }


/* ------------- Line edits and list views ---------------*/

QLineEdit, QListView {
    color: #808080;
	font: 18px DinPro-medium;
    selection-color: white;   
    border: 2px groove white;
    border-radius: 4px;
}


/* ---------- Sliders ------------*/

QSlider::groove:horizontal {
border: 1px solid rgb(  191, 191, 191 );
background: white;
height: 10px;
border-radius: 5px;
}

QSlider::sub-page:horizontal {
background: rgb( 51, 51, 51 );
border: 1px solid rgb( 51, 51, 51 );
height: 10px;
border-radius: 6px;
}

QSlider::add-page:horizontal   /* to the right of the handle */
{
background: rgb(  191, 191, 191  );
border: 1px solid rgb(  191, 191, 191  );
height: 10px;
border-radius: 6px;
}

QSlider::handle:horizontal {
  image: url(:images/handle_normal.png);
  margin: -15px 0;
    background:   rgb( 128, 128, 128 ); 
}

QSlider::handle:horizontal:hover {
	image: url(:images/handle_rollover.png);
    margin: -15px 0;
    background:   rgb( 128, 128, 128 ); 
}

QSlider::sub-page:horizontal:disabled {
    background:   rgb(  191, 191, 191  );
    border-color:  rgb(  191, 191, 191  );
border-radius: 6px;
}

QSlider::add-page:horizontal:disabled {
    background:    rgb(  191, 191, 191  );
    border-color:  rgb(  191, 191, 191  );
border-radius: 6px;
}

QSlider::handle:horizontal:disabled {
	image: url(:images/handle_disabled.png);
background:  rgb( 128, 128, 128 );
border: 1px solid  rgb(  128, 128, 128  );
border-radius: -15px;
}

/* --------- Check Boxes ----------*/

QCheckBox {
    font: 20px DINPro-medium;
	spacing: 5px;
}

QCheckBox::indicator {
	border:   rgb( 128, 128, 128 );
    background:   rgb( 128, 128, 128 );
	width: 20px;
	height: 20px;
	margin-top: 4px;
}

QCheckBox::indicator:unchecked {
	image: url(:images/checkBox_normal.png);
}

QCheckBox::indicator:checked {
	image: url(:images/checkBox_active.png);
}

QCheckBox::indicator:disabled {
	image: url(:images/checkBox_disabled.png);
}

/* --------- Radio Buttons ----------*/

QRadioButton {
	spacing: 5px;
}

QRadioButton::indicator {
    border:   rgb( 128, 128, 128 );
    background:   rgb( 128, 128, 128 ); 
    width: 20px;
    height: 20px;
    margin-top: 4px;
}

QRadioButton::indicator:unchecked {
	image: url(:images/checkBox_normal.png);
}

QRadioButton::indicator:checked {
	image: url(:images/checkBox_active.png);
}

QRadioButton::indicator:disabled {
	image: url(:images/bt_radio_disabled.png);
}

/* ----------  Other ---------- */

QDialogButtonBox
{
   button-layout: 1;
}

QWidget#backgroundWidget
{
     border-radius: 5px;
     background: rgb( 128, 128, 128 );
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="backgroundWidget" native="true">
     <property name="font">
      <font>
       <family>DinPRO-regular</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="titlebarLabel">
        <property name="styleSheet">
         <string notr="true">QLabel
{
	background: rgb(51, 51, 51 );
    font: 16pt &quot;DINPro-Medium&quot;;
    color: white;
    padding-left: 15px;
}</string>
        </property>
        <property name="text">
         <string>Engineering Dialog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxVoa">
        <property name="title">
         <string>Variable Optical Atenuation (VOA)</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>40</number>
         </property>
         <property name="topMargin">
          <number>40</number>
         </property>
         <item>
          <widget class="QLCDNumber" name="lcdNumberVOA">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Filled</enum>
           </property>
           <property name="value" stdset="0">
            <double>66.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderVoa">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="sliderPosition">
            <number>66</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxLaser">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Laser Power</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>40</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>40</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBoxLaserOn">
           <property name="text">
            <string>On</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxLaserOff">
           <property name="text">
            <string>Off</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxMotor">
        <property name="title">
         <string>Motor Speed[%]</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>40</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>40</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLCDNumber" name="lcdNumberMotorSpeed">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="digitCount">
            <number>4</number>
           </property>
           <property name="mode">
            <enum>QLCDNumber::Dec</enum>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Filled</enum>
           </property>
           <property name="value" stdset="0">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderMotorSpeed">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>600</number>
           </property>
           <property name="maximum">
            <number>3000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>100</number>
           </property>
           <property name="sliderPosition">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>500</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxRecorder">
        <property name="title">
         <string>Record or Playback</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>40</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <item row="2" column="2">
          <widget class="QCheckBox" name="checkBoxPlayback">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Playback</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelSpeed">
           <property name="text">
            <string>Playback Speed:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButtonLoad">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Load from file</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="labelFramesForPlayback">
           <property name="text">
            <string>Frame for playback: 0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButtonSave">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Save to file</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="labelStat">
           <property name="text">
            <string>TBD recording or playback statistics</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonFile">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Select File</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>120</height>
            </size>
           </property>
           <property name="text">
            <string>C:/Avinger_System/frameRef.data</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QSlider" name="sliderPlaybackSpeed">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
           <property name="sliderPosition">
            <number>500</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButtonSingleStep">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Single Step</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButtonSaveData">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Save the Data</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sliderVoa</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberVOA</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMotorSpeed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMotorSpeed</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
